job-service-client/pom.xml

```diff
@@ -34,13 +34,10 @@
         <relativePath/>
     </parent>
     
-    <organization>
-        <name>EntIT Software LLC, a Micro Focus company</name>
-        <url>https://www.microfocus.com</url>
-    </organization>
```

Comment:
I was wrong about the other one but this one is inheriting from `swagger-restapi-client-base::1.14.0-1` whose organization is `Hewlett Packard Enterprise Development LP`.